vendor/hal_sensors_default.te

```diff
@@ -11,3 +11,4 @@ allow hal_sensors_default persist_file:dir search;
 allow hal_sensors_default persist_sensors_file:dir create_dir_perms;
 allow hal_sensors_default persist_sensors_file:file create_file_perms;
 
+dontaudit hal_sensors_default kernel:system module_request;
```

Comment:
Please confirm this is ok to do. It follows crosshatch sepolicy, but might be needed. Please weigh in @jerpelea

Reply:
Dropped